common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/model/LoginModel.java

```diff
@@ -76,4 +143,228 @@ public String getSessionId() {
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_STOP);
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_PAUSE);
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_FREEZE);
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_RESUME);
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_SHUTDOWN);
+        methods.add(PERMISSION_SCHEDULER_FRONTEND_KILL);
+        return methods;
+    }
+
+    public List getRmTabsPermissionMethods() {
+        List<String> methods = new ArrayList<>();
+        methods.add(PERMISSION_RM_EXECUTE_SCRIPT);
+        methods.add(PERMISSION_RM_GET_RM_THREAD_DUMP);
+        methods.add(PERMISSION_RM_GET_NODE_THREAD_DUMP);
+        return methods;
+    }
+
+    public List getRmNodesPermissionMethods() {
+        List<String> methods = new ArrayList<>();
+        methods.add(PERMISSION_RM_LOCK_NODES);
+        methods.add(PERMISSION_RM_UNLOCK_NODES);
+        methods.add(PERMISSION_RM_REMOVE_NODES);
+        return methods;
+    }
+
+    public boolean userHasPermissionToStartJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_START) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_START);
+    }
+
+    public boolean userHasPermissionToStopJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_STOP) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_STOP);
+    }
+
+    public boolean userHasPermissionToFreezeJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_FREEZE) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_FREEZE);
+    }
+
+    public boolean userHasPermissionToResumeJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_RESUME) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_RESUME);
+    }
+
+    public boolean userHasPermissionToPauseJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_PAUSE) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_PAUSE);
+    }
+
+    public boolean userHasPermissionToShutDownJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_SHUTDOWN) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_SHUTDOWN);
+    }
+
+    public boolean userHasPermissionToKillJob() {
+        return sessionPermissions.containsKey(PERMISSION_SCHEDULER_FRONTEND_KILL) &&
+               sessionPermissions.get(PERMISSION_SCHEDULER_FRONTEND_KILL);
+    }
+
+    public boolean userHasPermissionToLockNodes() {
+        return sessionPermissions.containsKey(PERMISSION_RM_LOCK_NODES) &&
+               sessionPermissions.get(PERMISSION_RM_LOCK_NODES);
+    }
+
+    public boolean userDoesNotHavePermissionToLockNodes() {
+        return sessionPermissions.containsKey(PERMISSION_RM_LOCK_NODES) &&
+               !sessionPermissions.get(PERMISSION_RM_LOCK_NODES);
+    }
```

Comment:
Is it usefull to have a method for both conditions ? Cant we work with only `userHasPermissionToLockNodes()` ?

(same for others below)